hs_core/models.py

```diff
@@ -4198,7 +4198,24 @@ def _create_person_element(self, etree, parent_element, person):
         if person.name:
             hsterms_name = etree.SubElement(dc_person_rdf_Description,
                                             '{%s}name' % self.NAMESPACES['hsterms'])
-            hsterms_name.text = person.name
+
+            if "," in person.name:
```

Comment:
This whole block can be simplified to 
```
if "," in name:
    last_first = person.name.split(",")
    hsterms_name.text = "{} {}".format(last_first[1].strip(), last_first[0].strip()).strip()
else:
    hsterms_name.text = person.name
```

Reply:
It'd be nice to drop a comment in here that explains why we're looking for a "," in person.name and what that format looks like with an example.

Reply:
Better yet, we could send this value through the new filter `name_without_commas`